packaging/test_irods_client_rest_cpp.py

```diff
@@ -452,3 +453,19 @@ def test_zone_report(self):
 
             assert(js0 == js1)
 
+    def test_changing_passwords_is_supported__issue_99(self):
+        # Show that the user can execute commands without error.
+        self.user.assert_icommand(['ils', '-ld'], 'STDOUT', [self.user.session_collection])
+
+        # Show that changing the user's password will require them to reauthenticate.
+        token = irods_rest.authenticate('rods', 'rods', 'native')
+        new_password = 'rods'
```

Comment:
new password same as the old?

Reply:
that is the new password for the user inherited by the test.

the user's original password is 'apass'. see class name.

Reply:
oh, so let's change it to be something that is unique from the default admin password.     make it 'apassnew' or something that is self-describing.